### PR TITLE
AN-1846: implementation of Alt Text proposal

### PR DESCRIPTION
**Pull request by cmmovius**
Alt Text implementation of proposal from PR #189

### CODE REVIEW

File: src/main/resources/schema/ans/0.8.1/traits/trait_alt_text.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.8.1/traits/trait_alt_text.json",
+  "title": "Alt Text",
+  "description": "A phrase to describe the contents of an image to users.",
```
Comment (chadmoone) [inline src/main/resources/schema/ans/0.8.1/traits/trait_alt_text.json]: Sorry I didn't notice this during the proposal portion, but should we also specify/clarify here that this is used for accessibility?

Comment (gengel) [reply src/main/resources/schema/ans/0.8.1/traits/trait_alt_text.json]: Would it be sufficient to say this is the direct ANS equivalent of the HTML `alt` attribute?
https://html.spec.whatwg.org/#attr-img-alt

Comment (cmmovius) [reply src/main/resources/schema/ans/0.8.1/traits/trait_alt_text.json]: Good suggestions. I will update this to description to make it more clear that it is for accessibility and that it is "the direct ANS equivalent of the HTML alt attribute".

Comment (cmmovius) [reply src/main/resources/schema/ans/0.8.1/traits/trait_alt_text.json]: @chadmoone @gengel I have updated the description accordingly

**Pull request merged**